index.html

```diff
@@ -159,58 +159,77 @@ <h3>Leverage the power of open source in the field</h3>
       <span class="glyphicon glyphicon-chevron-right"></span>
       <span class="sr-only">Next</span>
     </a>
-
-    <a href="https://www.opengis.ch/2022/11/03/best-of-swiss-enterprise-app-award-for-qfield/" target="_blank"
-      style="display:block; position:absolute;top:20px;right:25px;width:140px;">
-      <img src="images/bosa2022gold.png" alt="Best of Swiss Enterprise App-Award 2022" style="width:140px;filter:drop-shadow(4px 4px 8px #000000)">
-    </a>
   </div>
 
   <!-- Section #2: claim -->
-  <section class="section-spacing text-center">
-    <h1>The most powerful and efficient way to manage your data on-the-go</h1>
+  <section class="section-spacing text-center" style="margin-bottom: 25px;">
+    <h2>The most powerful and efficient way to <span style="color: #80cc28">manage your data</span> on-the-go</h2>
```

Comment:
I like the green highlight but we already rely on underline-less green text as hyperlinks, which means I think most users will feel disoriented not being able to click there:

![image](https://github.com/user-attachments/assets/bc044688-f9bb-490d-b627-7ca7f9f84630)

More green hyperlinks:

![image](https://github.com/user-attachments/assets/98d3fdc2-7682-4a85-bc8e-4fb8d84f00b8)

:)